config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
```

Comment:
- [ ] Hello, kindly make sure to update your route to include routing for greeting controller api, see below code as an example

```suggestion
 namespace :api do
    namespace :v1 do
      resources :greetings, only: :index
    end
  end
```